src/connector.rs

```diff
@@ -616,6 +626,11 @@ impl EthConnectorContract {
     pub fn set_paused_flags(&mut self, args: PauseEthConnectorCallArgs) {
         self.set_paused(args.paused_mask);
     }
+
+    /// Return metdata
+    pub fn get_metadata(&self) {
+        sdk::return_output(&self.metadata.try_to_vec().unwrap()[..]);
```

Comment:
Do we need to return this as json (not Borsh)?

Reply:
It's Borsh

Reply:
I know we are currently returning it as Borsh. I am asking whether we should be returning it as json for compliance with what other contracts are expecting.

Reply:
We discussed it with @joshuajbouw 
According to [Metadata](https://github.com/near/NEPs/blob/master/specs/Standards/FungibleToken/Metadata.md) there is no concrete restrictions.

Reply:
This seems like an oversight in the spec.

The whole point of this PR is to be compliant with what other contracts (e.g. `ref.finance`) are expecting and I worry that they might be expecting json.

For example the wrapped NEAR FT returns json

```
$ NEAR_ENV=mainnet ./bin/near view wrap.near ft_metadata
View call: wrap.near.ft_metadata()
{
  spec: 'ft-1.0.0',
  name: 'Wrapped NEAR fungible token',
  symbol: 'wNEAR',
  icon: null,
  reference: null,
  reference_hash: null,
  decimals: 24
}
```

Reply:
Indeed `ref.finance` does assume metadata is returned via json: [`ftGetTokenMetadata`](https://github.com/ref-finance/ref-ui/blob/main/src/services/ft-contract.ts#L59) calls into [`ftViewFunction`](https://github.com/ref-finance/ref-ui/blob/main/src/services/ft-contract.ts#L24) which calls into [`viewFunction` (in `near-api-js`)](https://github.com/near/near-api-js/blob/master/src/account.ts#L499) with the default parser, [which is json](https://github.com/near/near-api-js/blob/master/src/account.ts#L503).

Reply:
We found with @joshuajbouw that reference FT implementation does not compline NEP-141.
So we should discuss it publicly.